pkgs/applications/misc/jquake/default.nix

```
@@ -14,10 +16,9 @@ stdenv.mkDerivation rec {
   sourceRoot = ".";
 
   postPatch = ''
-    # JQuake emits a lot of debug-like messages in console, but I
-    # don't think it's in our interest to void them by default. Log them at
-    # the appropriate level.
-    sed -i "/^java/ s/$/\ | logger -p user.debug/" JQuake.sh
+    # JQuake emits a lot of debug-like messages on stdout. Either drop the output
+    # stream entirely or log them at 'user.debug' level.
+    sed -i "/^java/ s/$/ ${if logOutput then "| logger -p user.debug" else ''> \/dev\/null''}/" JQuake.sh
```

```suggestion
    sed -i "/^java/ s/$/ ${if logOutput then "| logger -p user.debug" else "> /dev/null"}/" JQuake.sh
```
or does this need to be escaped?

---

Double quotes elide the `\`, which is needed by sed to escape the `/`

---

Oh but wait, I forgot to check if `\\` results to `\`, duh

---

It indeed does, silly me. Committing the prettier version now.